dipy/denoise/noise_estimate.py

```diff
@@ -260,11 +260,33 @@ def estimate_sigma(arr, disable_background_masking=False):
         If True, uses all voxels for the estimation, otherwise, only non-zeros
         voxels are used. Useful if the background is masked by the scanner.
 
+    N : int, default 1
+        Number of coils of the receiver array. Use N = 1 in case of a SENSE
+        reconstruction (Philips scanners) or the number of coils for a GRAPPA
+        reconstruction (Siemens and GE). See [1] for more information.
+
     Returns
     -------
     sigma : ndarray
         standard deviation of the noise, one estimation per volume.
+
+    Reference
+    -------
+    [1] Koay, C. G., & Basser, P. J. (2006). Analytically exact correction
+    scheme for signal extraction from noisy magnitude MR signals.
+    Journal of Magnetic Resonance), 179(2), 317–22.
+
+    Note
+    -------
+    This function is the same as manually taking the standard deviation of the
+    background and gives one value for the whole 3D array. Since it was
+    developped for T1 imaging. It is expected to perform ok on diffusion MRI data,
```

Comment:
I think this was meant to be one sentence, and there's a small typo in here ("developped"). It's also a bit vague, so might not help users that much. How about the following: 

```
This function implements equation 7 in [Koay2006]_: it applies a coil-number dependent correction factor to the standard deviation of the background and  produces a single value for the input volume, or 4D volume series. Since this formulation was developed for T1 imaging, it could work well for diffusion MRI, but take note: when used for denoising, this method may oversmooth some regions and leave others un-denoised. This is particularly true for cases in which the noise profile is not uniform across the volume. Consider using :func:`piesno` if visual examination  reveals non-uniform denoising performance.
```
As an aside, the Notes section should precede the "References" section.